specs/babbage/transaction.cddl

```diff
@@ -41,7 +41,7 @@ alonzo_tx_out = [
 ]
 babbage_tx_out = {
     0 : address,      ; @name address
-    1 : value,        ; @name value
```

Comment:
why value -> amount?

Reply:
For consistency, to match the name used in the other 2 variants.